### ISSUE TITLE
codesign application

### ISSUE THREAD

**Issue by matryer** [matryer/xbar#683]
Need to sort out code signing once and for all. We'll prioritise this, as we think it will fix lots of other things.

**Comment by gitrobert42** [comment]
Yes please! I actually came here to request that you sign the code. There are a couple reasons I would like it:
1. Unsigned codes throws a ton of error noise into the system.log. 
2. I would feel a lot less nervous about having it installed. 

I understand it costs $100 to get an Apple Developer license. If you could prioritize this issue, I would be happy to donate the $100.

**Comment by matryer** [comment]
@gitrobert42 thank you :) We're working on this. I have a developer license already, the trouble is making the tooling behave.

I feel a late night / early morning session with @leaanthony coming on :)

**Comment by danb-optimizely** [comment]
Any update on this?

**Comment by gimbo** [comment]
I'm currently unable to get xbar running on Big Sur 11.3.1, I think because of this issue. When I try to run it, I just get this:

![Screenshot 2021-06-01 at 14 19 21](https://user-images.githubusercontent.com/133770/120330182-7d923d00-c2e4-11eb-9703-3378836d2326.png)

This is happening with both the version download from [here](https://github.com/matryer/xbar/releases/tag/v2.0.42-beta) and via homebrew (both at 2.0.42-beta).

**Comment by leaanthony** [comment]
Please check latest version

**Comment by ZBiener** [comment]
Is code signing supposed to work in the latest version? I still get the same errors as reported on issue #681. Just want to make sure it isn't something on my end now. THanks.
